core/src/main/java/software/amazon/awssdk/core/interceptor/SdkExecutionAttributes.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.core.interceptor;
+
+import software.amazon.awssdk.annotations.ReviewBeforeRelease;
+import software.amazon.awssdk.core.RequestOverrideConfig;
+import software.amazon.awssdk.core.ServiceAdvancedConfiguration;
+import software.amazon.awssdk.core.runtime.auth.Signer;
+
+/**
+ * Contains attributes attached to the execution. This information is available to {@link ExecutionInterceptor}s and
+ * {@link Signer}s.
+ */
+public interface SdkExecutionAttributes {
+
+    /**
+     * The key under which the request config is stored.
+     */
+    @ReviewBeforeRelease("RequestConfig feels pretty internal. Can we just expose parts of it?")
+    ExecutionAttribute<RequestOverrideConfig> REQUEST_CONFIG = new ExecutionAttribute<>("RequestConfig");
```

Comment:
This comment doesn't seem to apply anymore since RequestOverrideConfig is a public class.